articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+4. **Set up target environment** Select your Azure account, Azure storage account and network that you want to use. During replication, VM data is copied to Azure storage. After failover, Azure VMs are joined to the specified network.
+
+5. **Enable replication** Configure replication settings, and enable replication for VMs. The Mobility service will be installed on a VM when replication is enabled. Site Recovery performs an initial replication of the VM, and then ongoing replication begins.
+
+6. **Run a disaster recovery drill** After replication is up and running, you verify that failover will work as expected by running a drill. To initiate the drill, you run a test failover in Site Recovery. The test failover doesn't impact your production environment.
+
+## Account Prerequisites
+
+For Azure site recovery to work you will need a Public Azure subscription as you will be deploying VM's onto this environment and isn't provided by UKCloud.
+
+## Configuration server prerequisites
+
+When setting up the Configuration server VM, it will need the following requirements for physical server replication.
+
+### Hardware Settings
+
+|Device                                    |Details               |
+|---------                                 |---------             |
+|CPU                                       | > 8 cores            |
+|RAM                                       | > 16GB               |
+|No of Disks                               | 3 Including OS disk  |
+|OS Disk space                             | > 200gb              |
+|Free Disk Space for process server cache  | > 600gb              |
+|Free Disk Space for retention disk        | > 600gb              |
+
+### Software Settings
+
+|Software                                  |Details               |
+|---------                                 |---------             |
+|OS                                        | Windows Server 2012 R2 / 2016 |
+|OS system locale                          | English (en-us)      |
+|IP address type                           | Static               |
+
+**Failure to have a static IP can cause issues during failover.**
+
+### Network access settings for configuration server
+
+* Firewall Ports - Allow ports 443 and 9443, both inbound and outbound (these are used during failover). It may be useful to set up a single **Network Security Group** for all the VMs in the resource group.
+
+* In the Windows Firewall on the configuration server, allow **File and Printer Sharing**, and **WMI**.
+
+    1. To do this, run wf.msc to open the Windows Firewall console.
+
+    2. Right select **Inbound Rules** > **New Rule**. Select Predefined, and choose File and Printer sharing from the list.
+
+    3. Complete the wizard, select to allow the connection > Finish.
+
+    4. Find the firewall rules related to WMI (**Windows Management Instrumentation**) and enable them.
+
+    5. For domain computers, you can use a GPO to do this.
```

Comment:
If this is related to the firewall rules bit (rather than the procedure as a whole), don't put it in a separate step

Reply:
done